source/language_service/src/completion/tests.rs

```diff
@@ -2083,19 +2083,6 @@ fn path_segment_before_glob() {
     );
 }
 
-#[test]
```

Comment:
This is a test for something that you are working on implementing, right? It would be better if we just ignored the test for now and cited a reason being that this functionality is in progress, rather than removing the test.

Reply:
No -- I just thought this test case didn't make sense anymore. After the strict validation I added in the parser, this test broke. I don't think it's worth any extra effort to support completions inside this syntactically invalid construct.